course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -122,14 +288,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(e1 + 5.0)"
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.1e"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As we've just seen, we have the ability to update the cube's data directly. Whenever we do this though, we should be mindful of updating appropriate metadata on the cube:"
+    "An even simpler example of broadcasting is doing arithmetic between a cube and a scalar value.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Exercise: </font></b>\n",
+    "    <p>What happens if you add <b><font face='courier' color='black'>5.2</font></b> to the <b><font face='courier' color='black'>e1</font></b> cube ?</p>\n",
```

Comment:
Perhaps explicitly say `the scalar value`. i.e. ``What happens if you add the scalar value 5.2 to the e1 cube ?````
When I first read this I wondered what "5.2" was, whether it had been defined earlier but I forgot about it.